### ISSUE TITLE
After converting the model to TFLite, there is a significant loss in accuracy.

### ISSUE THREAD

**Issue by panhu** [tensorflow/tensorflow#62709]
### 1. System information

-  Linux Ubuntu 16.04
- TensorFlow 2

### 2. Code

converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()

 with open('model.tflite', 'wb') as f:
   f.write(tflite_model)

#### Option A: Reference colab notebooks

Hello, may I ask why there is a significant decrease in accuracy when using tflite for inference after converting the h5 model to tflite? I did not use quantification.

Thanks!

**Comment by panhu** [comment]
My models are all basic convolutional、GRU and deconvolution layers。

**Comment by LakshmiKalaKadali** [comment]
Hi @panhu,

Please make sure that the model layers, activation functions, padding  and other parameters are configured properly and also check if the model consists of any unsupported issues. If the problem still persists, please share reproducible code for further help. 
Thank You

**Comment by panhu** [comment]
Thanks, When I quantitatively perceive the model, I make an error "Runrime Error:Layer tf_op_layer_Reshape:<class 'keras.src.engine.base_layer.TensorFlowOpLayer'> is not supported. you can quantize this layer by passing a 'tfmot.quantization.keras.QuantizeConfig' instance to the quantize_annotate_layer" API.
